### ISSUE TITLE
How to create an downstream object detection task 

### ISSUE THREAD

**Issue by letdivedeep** [shuuchen/DetCo.pytorch#2]
Hi @shuuchen 

Thanks for the wonderful and ready-to-use repo. I have few questions, to just improve my understanding of the SSL approch.

As we use Detco for the pre-text tasks on certain unlabeled datasets like(coco) using the ResNet 50 Architecture.  Onces we have the pre-trained model how do we set it for the downstream object detection task :

-  Will this be with Supervision => where we have the images and the related bbox information about it

- Architecture => if for the DetCo pre-trained model I have used Resnet 50 and for the Downstream object detection with labels I want to use mobilenetV2 .. is it possible ? or it should be resnet 50 itself for downstream task

**Comment by shuuchen** [comment]
Thank you.

・Yes. You can use the pre-trained backbone for various supervised downstream tasks by appending the corresponding prediction heads to the backbone. 
・It should be the resnet50 itself for the downstream task. If you want to use mobilenetV2 for downstream task, you have to use mobilenetV2 for pre-training.

**Comment by letdivedeep** [comment]
@shuuchen  thanks for the detailed answer it helps
